src/Compute/Compute/Generated/Models/PSVirtualMachineScaleSetVMInstanceView.cs

```diff
@@ -37,6 +37,7 @@ public partial class PSVirtualMachineScaleSetVMInstanceView
         public VirtualMachineHealthStatus VmHealth { get; set; }
         public BootDiagnosticsInstanceView BootDiagnostics { get; set; }
         public IList<InstanceViewStatus> Statuses { get; set; }
+        public string AssignedHost { get; }
         public string PlacementGroupId { get; set; }
 
```

Comment:
has this been tested? my previous knowledge was all the properties of PSObjects have to have a set method to be initialized

Reply:
yes the assignedhost property is being returned in the TestVMSSAssignedhost. that's all its needed for now.

Reply:
seems to be working fine in the getvmsstestassignedhost. 
https://github.com/Azure/azure-powershell/pull/12588/files#diff-43fd27334ca98d2333cf8bee54f9a404R2578

Reply:
doesnt look like it's working for me when I test manually

PS C:\Repositories\PSmaster\artifacts\Debug\Az.Compute> $vmss1[0].instanceview

PlatformUpdateDomain      :
PlatformFaultDomain       : 2
RdpThumbPrint             :
VmAgent                   : Microsoft.Azure.Management.Compute.Models.VirtualMachineAgentInstanceView
MaintenanceRedeployStatus :
Disks                     : {MyVmss_MyVmss_2_OsDisk_1_11a4f26f3d2049b1b3e410989342b014}
Extensions                : {Microsoft.Azure.Geneva.GenevaMonitoring}
VmHealth                  :
BootDiagnostics           :
Statuses                  : {Microsoft.Azure.Management.Compute.Models.InstanceViewStatus,
                            Microsoft.Azure.Management.Compute.Models.InstanceViewStatus}
AssignedHost              :
PlacementGroupId          : e6dd9b98-7d04-41b2-9e86-114826f78d62

Reply:
@haagha

Reply:
Were you able to add hostgroupId in newazvmss? it gives me an error when i run manually

Reply:
yeah i get weird behavior. Most of the times when I tried to run new-azvmss with HostGroupId, I got error saying "Fault Domain Count on vmss has to be equal or less than Fault Domain Count on the HostGroup". But Right now Fault Domain count is not a parameter on new-azvmss so i couldn't set it, but we got approval to add it here https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/705
If you want to add it. I think Adam already added it for VMO feature @Sandido 

But one time new-azvmss did create it without giving me that error, but I am not able to see the instanceView of the VM Profile. 

If you want to check it out, I have the HostGroup, Host, and Vmss created in theo_test resourcegroup